querybook/webapp/components/DataTableViewColumn/DataTableColumnCard.tsx

```diff
@@ -71,3 +81,57 @@ export const DataTableColumnCard: React.FunctionComponent<IProps> = ({
         </div>
     );
 };
+
+interface IDataTableColumnCardNestedTypeProps {
+    complexType: ComplexType;
+}
+export const DataTableColumnCardNestedType: React.FunctionComponent<
+    IDataTableColumnCardNestedTypeProps
+> = ({ complexType }) => {
+    const hasChildren = complexType.children?.length > 0;
+    const [expanded, setExpanded] = React.useState(false);
+
+    const rowProps = {
+        className: 'flex-row',
+    };
+
+    if (hasChildren) {
+        rowProps['onClick'] = () => setExpanded(!expanded);
+        rowProps['aria-label'] = expanded
+            ? 'click to collapse'
+            : 'click to expand';
+        rowProps['data-balloon-pos'] = 'down-left';
+    }
+
+    return (
+        <div className="DataTableColumnCardNestedType">
+            <div {...rowProps}>
+                {hasChildren && (
+                    <IconButton
+                        icon={expanded ? 'Minus' : 'Plus'}
+                        size="16"
+                        noPadding={true}
+                        className="expand-icon"
+                    />
+                )}
+
+                <StyledText
+                    color="light"
+                    className={`column-type mr12 ${
+                        !hasChildren && 'nested-indent'
+                    }`}
+                >
+                    {complexType.type}
+                </StyledText>
+                <AccentText weight="extra">{complexType.key}</AccentText>
```

Comment:
instead of displaying it as `type  key`, I'd prefer to have it same as the `ColumnRow`, which is `key type`, which I think is more reasonable.